frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.2vms.ts

```diff
@@ -0,0 +1,29 @@
+import { withResource } from '@console/shared/src/test-utils/utils';
```

Comment:
I'm not sure if there is a reason to put this in a separate file, it's nearly identical to the original one.
I think it would be better to keep these in the original file

Reply:
After conversation with Radim we decided to leave it as is, resolving